controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -174,6 +178,37 @@ func (w *Workload) UpdateKubeletConfigMap(ctx context.Context, version semver.Ve
 		return err
 	}
 
+	// In order to avoid using two cgroup managers on the same machine, cgroupfs and systemd cgroup, starting from
```

Comment:
```suggestion
	// Kubeadm manages the kubelet using the systemd init system. In order to avoid using two cgroup managers on the same machine (cgroupfs and systemd), starting from
```

Reply:
updated according vince suggestion.